test/format/verify.c

```diff
@@ -234,6 +235,20 @@ table_verify_mirror(
       table->id, checkpoint == NULL ? "" : checkpoint, checkpoint == NULL ? "" : " checkpoint ");
     trace_msg(session, "%s: start", buf);
 
+    /*
+     * If we are not reading from a checkpoint, start a cursor to pin a page to ensure we never
+     * refresh our snapshot in verification.
+     */
+    if (checkpoint == NULL) {
```

Comment:
Should this just be an else case for the earlier conditional on the `checkpoint` arg?

Reply:
I want to add the code after the previous trace message.